source/_patterns/01-molekyler/05-lister/20-liste-unummerert-expandable.json

```diff
@@ -0,0 +1,403 @@
+                    "icon-prefix":"reg",
+                    "icon-class":"reg-circle-exclamation",
+                    "icon-extra-class":"notificationIcon"
+                  }
+                }, 
+                { 
+                  "column-text":{ 
+                    "classes":"a-js-sortValue deleteOrdelegateText",
+                    "text":"Ber om Tilgang"
+                  }
+                }
+              ]
+            }
+        ],
+        "expandable-section":false
+      }, 
+      { 
+        "additional-classes":"a-list-parentRightHolder",
+        "onclick":false,
+        "expandedLink":{
+          "data-toggle":"",
+          "target-Link":"#"
+        },
+        "columns":[ 
+            { 
+              "additional-classes":"col-2 col-sm-1 col-lg-2",
+              "onclick":false,
+              "column-items":[ 
+                  { 
+                    "column-icon":{ 
+                        "icon-prefix":"ai",
+                        "icon-class":"ai-private",
+                        "icon-extra-class":""
+                    }
+                  }
+              ]
+            },
+            { 
+              "additional-classes":"col-6 col-sm-4 col-lg-4",
+              "onclick":false,
+              "column-items":[ 
+                  { 
+                    "column-text":{ 
+                        "classes":"a-js-sortValue",
+                        "text":"Ovre Petter Holt"
+                    }
+                  }
+              ]
+            },
+            { 
+              "additional-classes":"col-md-3 d-none d-md-block col-lg-3",
+              "onclick":false,
+              "column-items":[
+                { 
+                  "column-text":{ 
+                      "classes":"a-js-sortValue",
+                      "text":"Daglig leder"
+                  }
+                }
+              ]
+            },
+            { 
+              "additional-classes":"col-4 col-sm-7 a-hiddenWhenSelected col-lg-3",
+              "onclick":false,              
+              "column-items":[ 
+                {
+                  "column-icon":{ 
+                    "icon-prefix":"",
+                    "icon-class":"",
+                    "icon-extra-class":"notificationIcon d-none"
+                  }
+                }, 
+                { 
+                  "column-text":{ 
+                    "link-class":"text-right a-fontBold collapsed pr-0 pr-md-5",
+                    "classes":"a-btn-blue-noUnderline a-js-sortValue  a-fontBold expandText deleteOrdelegateText",
+                    "text":"Gi eller fjern tilgang"
+                  }
+                }
+              ]
+            }
+        ],
+        "expandable-section":false
+      },
+      { 
+        "additional-classes":"a-list-parentRightHolder",
+        "onclick":false,
+        "expandedLink":{
+          "data-toggle":"collapse",
+          "target-Link":"#nested-list-chiledRightHolders"
```

Comment:
hva er `chiled`

Reply:
det er skrivefeil, mente child. det er en rightholderlist for en organisasjon som ligger i første listen